### DISCUSSION TITLE
How to Donwload?

### DISCUSSION THREAD

**Discussion by Mourice326** [IceHacks/SurvivCheatInjector#800] [Q&A]
Can you tell me how how to download and put the zips in use. On Windows. I kind of forgot how. 
Thanks 👍

**Comment by ghost** [comment]
you download it

**Reply by Mourice326** [reply]
I know that. What do you do after that is what i'm asking.

**Comment by Cat-man-2735** [comment]
Where is the download button? And can I get this on chrome?And is there a tutorial video?

**Reply by Mourice326** [reply]
Thats what im asking
